sunpy/util/decorators.py

```diff
@@ -182,3 +183,27 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class Appender(object):
```

Comment:
`Appender` is not a very descriptive name for this functionality. You want something that actually says what it is, i.e. something about docstrings.